src/ontology/mondo-ingest.Makefile

```diff
@@ -664,14 +664,19 @@ $(EXTERNAL_CONTENT_DIR)/nando-mappings.robot.tsv: $(MAPPINGSDIR)/mondo-nando.sss
 
 # OTAR / EFO
 .PHONY: external-content-efo
-external-content-efo: $(EXTERNAL_CONTENT_DIR)/mondo-efo.robot.owl $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.owl
+external-content-efo: $(EXTERNAL_CONTENT_DIR)/mondo-efo.robot.owl $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.owl $(EXTERNAL_CONTENT_DIR)/efo-proxy-merges.robot.owl
 
 $(EXTERNAL_CONTENT_DIR)/mondo-efo.robot.tsv:
 	wget "https://raw.githubusercontent.com/EBISPOT/efo/master/src/ontology/reports/mondo-efo.robot.tsv" -O $@
 
 $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.tsv:
 	wget "https://raw.githubusercontent.com/EBISPOT/efo/master/src/ontology/reports/mondo-otar-subset.robot.tsv" -O $@
 
+# Unfortunately, @matentzn cant find the orginal TSV file, but it is highly unlikely this file will ever change.
+
+$(EXTERNAL_CONTENT_DIR)/efo-proxy-merges.robot.owl:
+	echo "WARNING: $@" is currently manually curated!"
```

Comment:
Usually `robot.owl`s are created from `robot.tsv`s. It makes me wonder if the manually curated file was a`efo-proxy-merges.robot.tsv`? I do see that `efo-proxy-merges.robot.owl` says "Generated by the OWL API".

I suppose we don't expect the `robot.tsv` to ever change, so you are excluding it and simply including the `robot.owl` statically?
If so, then why is there a goal for it? Is it simply so that we see this warning appear in the log, and no other reason?

Reply:
> It makes me wonder if the manually curated file was aefo-proxy-merges.robot.tsv? I do see that efo-proxy-merges.robot.owl

Yes, see above. Probably.

> I suppose we don't expect the robot.tsv to ever change, so you are excluding it and simply including the robot.owl statically?

The TSV does not exist at all, see comment above.

> If so, then why is there a goal for it? Is it simply so that we see this warning appear in the log, and no other reason?

Yes, just to have a record. Its easier to debug it like this, and for a developer to see when the pipeline is updated.